app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide :title, t(".help_title") %>
+<h1><%= t ".heading" %></h1>
+
+<p>
+  <%= t ".description_title" %>
+  <%= link_to t("ruby_on_rails_help"), "http://www.railstutorial.org/help" %>
+  <%= t ".description_detail" %>
+  <%= link_to t("ruby_on_rails_book"), "http://www.railstutorial.org/book" %>
```

Comment:
with link such as `http://www.railstutorial.org/help` you should move it to settings.yml file and call this here.
You can research `gem config`

Reply:
done